### PR TITLE
add spec files

### PR DESCRIPTION
**Pull request by RubyDiver**

### CODE REVIEW

File: .idea/MonteCinema.iml [hunk 1]
```diff
@@ -47,6 +47,7 @@
     <orderEntry type="library" scope="PROVIDED" name="crass (v1.0.6, RVM: ruby-3.0.1) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="diff-lcs (v1.4.4, RVM: ruby-3.0.1) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="erubi (v1.10.0, RVM: ruby-3.0.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="factory_bot (v6.2.0, RVM: ruby-3.0.1) [gem]" level="application" />
```
Comment (w0st) [inline .idea/MonteCinema.iml:50]: IMHO you shouldn't keep your personal editor settings in the repository.

It's a good candidate for local or global `.gitignore` entry. I encourage you to read [this](https://dart.dev/guides/libraries/private-files#details)

Comment (wmaciejak) [reply .idea/MonteCinema.iml:50]: +1 👍

File: Gemfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ gem 'puma', '~> 5.0'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'factory_bot'
```
Comment (wmaciejak) [inline Gemfile]: Consider moving this file into `test` or `development and test` group.

File: spec/models/cinema_hall_spec.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe CinemaHall, type: :model do
+
+  it 'valid with correct info' do
```
Comment (wmaciejak) [inline spec/models/cinema_hall_spec.rb]: Remember that description in the `it` block should contain information about what this particular test is testing. I know that this file is under development, but it's an early stage of code review :)

**Comment by wmaciejak** [comment]
Except for a few comments, looks really cool. Consider adding a few specs to models to be sure that everything works fine :)

**Pull request merged**